qulice-maven-plugin/src/it/pmd-violations/src/main/java/com/qulice/foo/Violations.java

```diff
@@ -87,7 +84,7 @@ public static void print() {
     /**
      * Test class.
      */
-    private static final class Foo {
+    private final class Foo {
```

Comment:
@vkuchyn Why removing this static modifier here. qulice should not complain about it. Is it complaining here ?